servers.json

```diff
@@ -1226,7 +1226,9 @@
        "id": "cylonemc",
         "name": "CyloneMC",
         "addresses": [
-            "cylonemc.net"
+            "cylonemc.net",
+            "cylonemc.com",
+            "pvp.cylone.net"
```

Comment:
Please look over the checklist again.

Reply:
We don't own cylone.net only the pvp.cylone.net subdomain, so we don't want to match *.cylone.net

Reply:
> We don't own cylone.net only the pvp.cylone.net subdomain, so we don't want to match *.cylone.net

Okay, I'll get back to you in a bit.